packages/eslint-plugin/src/rules/keyword-spacing.ts

```diff
@@ -0,0 +1,293 @@
+import baseRule from 'eslint/lib/rules/keyword-spacing';
+import ESLINT_UTILS_KEYWORDS from 'eslint/lib/rules/utils/keywords';
+import {
+  TSESTree,
+  AST_TOKEN_TYPES,
+} from '@typescript-eslint/experimental-utils';
+import { isTokenOnSameLine } from '../util/astUtils';
+import * as util from '../util';
+
+export type Option = Partial<{
+  before: boolean;
+  after: boolean;
+}>;
+export type OverrideOptions = Partial<Record<string, Option>>;
+export type RootOption = Option & { overrides?: OverrideOptions };
```

Comment:
you shouldn't need to redefine these - just the `Options` type below should be enough
```suggestion
```